### DISCUSSION TITLE
Fooocus unable to recognize NVIDIA driver

### DISCUSSION THREAD

**Discussion by LostDaemon** [lllyasviel/Fooocus#3271] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/896a1c42-3ea4-4e9d-86dc-7dea1da965ad)

Container stops immideately with the error "RuntimeError: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed"

The only graphic card in the system is NVIDIA RTX 4070TI Super with most recent drivers. 
Its a normal desktop PC  (not a laptop), so no integrated graphics or anything like this.
I updated drivers when first faced this issue - no luck.
What's could gone wrong?

UPD: Looks like I managed to fix that step by using "docker run --gpus all"

But after that I faced another exception:
"RuntimeError: Unexpected error from cudaGetDeviceCount(). Did you run some cuda functions before calling NumCudaDevices() that might have already set an error? Error 500: named symbol not found"

**Comment by mashb1t** [comment, marked as answer]
Please follow the official instructions in https://github.com/lllyasviel/Fooocus/blob/main/docker.md or use the [docker-compose.yml](https://github.com/lllyasviel/Fooocus/blob/main/docker-compose.yml) file with `docker compose up -d`.
